pandas/_libs/window.pyx

```diff
@@ -1682,7 +1682,7 @@ def roll_window(ndarray[float64_t, ndim=1, cast=True] values,
     Assume len(weights) << len(values)
     """
     cdef:
-        ndarray[float64_t] output, tot_wgt, counts
+        float64_t[:] output, tot_wgt, counts
```

Comment:
```
pandas/_libs/window.cpp:43820:21: warning: ‘__pyx_pybuffernd_tot_wgt.__Pyx_LocalBuf_ND::diminfo[0].__Pyx_Buf_DimInfo::strides’ 
may be used uninitialized in this function [-Wmaybe-uninitialized]
43820 |   __Pyx_LocalBuf_ND __pyx_pybuffernd_tot_wgt;
```

Reply:
Are you saying this resolved that issue or introduced it?

If resolved change lgtm just want to clarify

Reply:
I should have put the comment one line up. The change fixes this warning, which occurs because of the original line.